Makefile

```diff
@@ -154,6 +154,13 @@ IS_CLANG_COMPILER = $(shell $(CC) --version 2>&1 | $(EGREP) -i -c "clang version
 ifeq ($(shell uname),Darwin)
 SHARED_EXT = dylib
 PREFIX = /usr/local
+
+	# add openssl headers for MacOS
+	ifeq ($(CRYPTO_ENGINE_PATH),openssl)
+		CFLAGS += -I$(PREFIX)/opt/openssl/include
+		LDFLAGS += -L$(PREFIX)/opt/openssl/lib
```

Comment:
@mnaza 
I can use $(PREFIX) here.

I can get this path by
```
➜  local brew --prefix openssl
/usr/local/opt/openssl
```

however it works when user installs openssl via homebrew